components/yaml/introduction.rst

```diff
@@ -127,20 +127,6 @@ error occurred:
 
 .. _components-yaml-dump:
 
-Objects for Mappings
```

Comment:
Yes, but wouldn't that better fit just above the "Using Objects for Maps" headline?